src/category_theory/opposites.lean

```diff
@@ -113,11 +113,11 @@ protected def unop (F : Cᵒᵖ ⥤ Dᵒᵖ) : C ⥤ D :=
   map := λ X Y f, (F.map f.op).unop }
 
 /-- The isomorphism between `F.op.unop` and `F`. -/
-def op_unop_iso (F : C ⥤ D) : F.op.unop ≅ F :=
+@[simp] def op_unop_iso (F : C ⥤ D) : F.op.unop ≅ F :=
```

Comment:
```suggestion
@[simps] def op_unop_iso (F : C ⥤ D) : F.op.unop ≅ F :=
```
would be better I think: I'd rather there are simp lemmas to decompose the definition rather than have it automatically unfolded

Reply:
done!